### ISSUE TITLE
[Firefox] Automatic updates no longer working

### ISSUE THREAD

**Issue by 5t3f4n** [gorhill/uBlock#299]
When 8.9.8.2 was released the AMO version was updated within one hour (!) of being released on GitHub. As of writing this, it's now been 4 days since 8.9.8.3 was released and 13 hours since 8.9.8.5 was released and still no updates past 8.9.8.2. What's changed?

**Comment by gorhill** [comment]
I did not release 0.9.8.3 to AMO when it came out, it was mostly a sort of emergency Chrome store update to give users the mean to opt out of disabling prefetching.

The Firefox-related code changes in 0.9.8.3 were not ready for a stable release -- and this was confirmed as a few quirks had to be fixed. I released this morning 0.9.8.5 to AMO. However I made a mistake and somehow ended up uploading 0.9.8.3 by mistake. When I realized this I deleted as fast as I could the wrongly uploaded package and re-uploaded the real latest sable release, 0.9.5.0.<sup>[1]</sup>

By the time I had deleted the wrong package and re-uploaded the correct one, I received a notice from AMO telling me that _"preliminary review request has been approved"_ for 0.9.8.3...

Then I immediately received another notice from an AMO reviewer that I should not have deleted 0.9.8.3, because:

> deleting previously reviewed versions can add a lot of time to the review process and your waiting time.

Summary, if I had not uploaded the wrong package, chances are 0.9.8.5 would be live now in the store. I wanted to save work to AMO reviewers by removing the obsolete version I uploaded by mistake and replace it with the latest stable one, and now it appears that preliminary review for this one is going to take time.

***

<sub>[1] I wanted to spare work for the AMO reviewers by removing a version which was obsolete.</sub>
